src/test/java/uk/gov/hmcts/ccd/domain/service/createcase/SubmitCaseTransactionTest.java

```diff
@@ -167,29 +209,48 @@ void shouldGrantAccessToCreator() {
     @DisplayName("should invoke callback")
     void shouldInvokeCallback() {
         submitCaseTransaction.submitCase(event,
-                                         caseType,
-                                         idamUser,
-                                         eventTrigger,
-                                         this.caseDetails,
-                                         IGNORE_WARNING);
+            caseType,
+            idamUser,
+            eventTrigger,
+            this.caseDetails,
+            IGNORE_WARNING);
```

Comment:
Previous formatting was correct and shouldn't have been changed

Reply:
Looks like my intellij formatter is being too helpful